### ISSUE TITLE
Browser Automatic filling don't work for <b-input type="password" />

### ISSUE THREAD

**Issue by chaosdogl** [buefy/buefy#792]
### Overview of the problem

**Buefy** version: 0.6.5
**Vuejs** version: 2.5.16
**OS/Browser**: macOS 10.13.4/Safari Version 11.1, Chrome Version 65.0.3325.18 

### Description
input component can't get password from Browser's autofill.
Inspect this component's attr with Vue devtool, bind value by v-model is "",
b-input(v-model="username") 's behavior is expected
This is a minify case: https://jsfiddle.net/22bc8zy4/
// only reproduction in real environment

### Expected behavior
Browser will auto fill b-input(type="password")

### Actual behavior
b-input's value === ''

**Comment by jtommy** [comment]
@warcryDoggie I think you should change default value of `autocomplete` prop

**Comment by warcryDoggie** [comment]
I took some time to open the buefy box, This problem may stem from Vue or Apple. 

This GIf demonstrates the strange behavior of compontents  

I add `alert` in src/compontents/input.vue onInput method
```js
            onInput(event) {
                alert('firing onInput')
                this.$nextTick(() => { this.newValue = event.target.value })
            }
```
![before](https://user-images.githubusercontent.com/16441901/39560150-2b423ac4-4ece-11e8-97a0-0209e27100fd.gif)
 onInput was triggered, Until I clicked and dom update occurred.

remove prevention
```js
          onInput(event) {
                alert('firing onInput')
                // this.$nextTick(() => { this.newValue = event.target.value })
                this.newValue = event.target.value
            }
```
![after](https://user-images.githubusercontent.com/16441901/39560373-1f0359da-4ed0-11e8-960e-6c2b23b7c272.gif)
Triggering onInput still requires additional updates, but strange behavior that password Automatically clear is avoided.

Some additional information:
* I tested The earliest code on an Andriod with unknown version, everything is normal.
* autocomplete="on" does not help this issue
